client/signup/steps/page-picker/shopping-cart-for-difm.tsx

```diff
@@ -199,9 +206,10 @@ export default function ShoppingCartForDIFM( { selectedPages }: { selectedPages:
 
 					<Total>
 						<div>{ translate( 'Total' ) }</div>
-						<div className="page-picker__value">{ total }</div>
+						<div className="page-picker__value">{ total }*</div>
 					</Total>
 				</LineItemsWrapper>
+				<div className="page-picker__disclaimer">*Final price will be calculated at checkout.</div>
```

Comment:
Good idea. I know you're still working on this, but don't forget to translate this string!